jib-cli/src/main/java/com/google/cloud/tools/jib/cli/Jar.java

```diff
@@ -150,6 +151,14 @@
           "Entrypoint for container. Overrides the default entrypoint, example: --entrypoint='custom entrypoint'")
   private List<String> entrypoint = Collections.emptyList();
 
+  @CommandLine.Option(
+      names = "--creation-time",
+      paramLabel = "<creation-time>",
+      description =
```

Comment:
Another option is to provide the default (e.g., 0?). That way, maybe `creationTime` can never be null?

Either way has pros and cons though.

Reply:
I did think about adding a default but I assumed this would be a little redundant since we already have a default value for this field in jib?

Reply:
Either is fine. At least we can document the default value in the option description.